### DISCUSSION TITLE
请问是否有研究生开题的开题报告与文献综述模板？

### DISCUSSION THREAD

**Discussion by IRandonation** [BITNP/BIThesis#563] [Q&A] [answered]
### 发生了什么？

请问是否可以更新文献综述与开题报告模板

### 问题复现的步骤（或者截图）

在看项目内容的时候看到有本科的开题报告，请问是否有研究生的开题报告与文献综述的latex模板

**Comment by fky2015** [comment, marked as answer]
> 在看项目内容的时候看到有本科的开题报告，请问是否有研究生的开题报告与文献综述的latex模板

本科的开题报告已经废弃，原因是本科生的开题报告可以通过在线平台填写表格，不再需要自行完成并提交 PDF。

基于类似的原因，我们也没有开发研究生的开题报告和文献综述：
研究生的这两个模板需要提交 Word 版本的电子存档到学院（至少是网安学院），因此学生无法绕过 Word。并且由于字数不是太多，此时选用 Word 更方便一些。
